docs/guides/setup/configuring_data_contexts/how_to_configure_credentials.md

```diff
@@ -21,47 +22,49 @@ If your Great Expectations deployment is in an environment without a file system
 
 <Prerequisites></Prerequisites>
 
-**Steps**
+## Steps
 
-1. Decide where you would like to save the desired credentials or config values - in a YAML file, environment variables, or a combination - then save the values. In most cases, we suggest using a config variables YAML file. YAML files make variables more visible, easily editable, and allow for modularization (e.g. one file for dev, another for prod).
+### 1. Decide where you would like to save the desired credentials or config values - in a YAML file, environment variables, or a combination - then save the values.
```

Comment:
Just a stylistic preference, but could these step titles be shortened and the rest of the text be placed on the line below? For example:
```suggestion
### 1. Save credentials and config

Decide where you would like to save the desired credentials or config values - in a YAML file, environment variables, or a combination - then save the values.
```

Reply:
Absolutely we can.  I think shorter headings work better, too.